nemo_megatron_launcher/launch_scripts/nemo_megatron/collections/eval_harness/evaluate.py

```diff
@@ -0,0 +1,462 @@
+import argparse
+import glob
+import json
+import logging
+import os
+import random
+import shutil
+import string
+import subprocess
+import sys
+import time
+from datetime import datetime
+from importlib import reload
+from typing import Union
+
+from lm_eval import base, evaluator, models, tasks, utils
+from omegaconf import OmegaConf
+
+import nemo.collections.nlp as nemo_nlp
+from nemo.collections.nlp.models.language_modeling.megatron_gpt_model import MegatronGPTModel
+from nemo.utils.get_rank import is_global_rank_zero
+
+reload(logging)
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)-15s | %(name)-7s | %(levelname)-8s: %(message)s")
+logger = logging.getLogger(__name__)
+
+
+def parse_args(parser_main):
```

Comment:
Why didn't we use Hydra with evaluate.py?

Reply:
These are ported from megatron-lm's eval-harness. we didn't update to use hydra haha